### PR TITLE
Lisa's changes to json files for socioecon drawer tools into main

### PR DESCRIPTION
**Pull request by k-doering-NOAA**
Updated information provided for FishEYE and SIFC, including contact info and authors

### CODE REVIEW

File: model_list_dir/SIFC.json [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
+  "documentation_link": "https://www.fisheries.noaa.gov/national/socioeconomics/social-indicator-supporting-information",
+  "citation": "NOAA Fisheries Office of Science and Technology. 2020. NOAA Fisheries Community Social Vulnerability Indicators (CSVIs). Version 4 (Last updated: September 20, 2023).", 
   "active_development": true,
   "noaa_internal": true,
   "uses_github_releases": false,
   "toolbox_drawers": ["Human Dimensions"], 
-  "keywords": ["Web App", "Data Exploration", "Socioeconomics"]
-}
\ No newline at end of file
+  "keywords": ["Web App", "Data Exploration", "Socioeconomics", "Indicators", "Fishing Engagement and Reliance"]
```
Comment (k-doering-NOAA) [inline model_list_dir/SIFC.json:16]: @lpfeif these are the current keywords being used on the toolbox: https://github.com/noaa-fisheries-integrated-toolbox/toolbox_web_templating/blob/main/json_analysis/keywords.txt

Do any of the existing one work? If you feel like a new one is needed, we can add it though.

Also, sorry there was no documentation on the keywords - I'll add some text to the readme!

File: model_list_dir/fisheye.json [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
   "tool_name": "Fisheries Economic Explorer",
+  "references": ["Harsch, M., L. Pfeiffer, E. Steiner, and M. Guldin. 2018. Economic Performance Metrics : An Overview of Metrics and the Use of Web Applications to Disseminate Outcomes in the U.S. West Coast Groundfish Trawl Catch Share Program. NOAA technical memorandum NMFS-NWFS 143.","Steiner, E., A. Vizek, M. Guldin, M. Krigbaum, and L. Pfeiffer. 2021. Evaluating the Economic Performance of the U.S. West Coast Groundfish Trawl Catch Share Program. NOAA Tech Memo NMFS-NWFSC 169."],
   "online_app_link": "https://connect.fisheries.noaa.gov/fisheye/fisheyelandingpage.html",
+  "documentation_link": "https://connect.fisheries.noaa.gov/fisheye_literature/Literature.html",
   "active_development": true,
   "noaa_internal": true,
   "uses_github_releases": false,
   "toolbox_drawers": ["Human Dimensions"],
-  "keywords": ["Web App", "Economics"]
+  "keywords": ["Web App", "Economics", "West Coast Groundfish Trawl Catch Share Program"]
```
Comment (k-doering-NOAA) [inline model_list_dir/fisheye.json:16]: same comment about keywords here!

**Comment by k-doering-NOAA** [comment]
@lpfeif, here are the preview pages: https://noaa-fisheries-integrated-toolbox.github.io/fit-dev/fisheye and https://noaa-fisheries-integrated-toolbox.github.io/fit-dev/SIFC.

One thing I noticed that needs to be fixed is that the contributors are not showing up. I'll need to push up a change to fix this.

File: model_list_dir/fisheye.json (hunk 2 above)
Comment (lpfeif) [reply model_list_dir/fisheye.json:16]: Ah, you can just delete the ones I added. Was trying to be more descriptive but didn't realize there was a list!

Comment (k-doering-NOAA) [reply model_list_dir/fisheye.json:16]: My fault for not telling you! Ok, I will remove them.

**Comment by k-doering-NOAA** [comment]
@lpfeif are you happy with how the preview pages look ( https://noaa-fisheries-integrated-toolbox.github.io/fit-dev/fisheye and https://noaa-fisheries-integrated-toolbox.github.io/fit-dev/SIFC)? 

If so, I think this is ready to merge in!

**Comment by lpfeif** [comment]
Both look great!

**Pull request merged**